setup.cfg

```diff
@@ -23,11 +23,27 @@ atomic=True
 # This ensures compatibility with pytest-pep8
 lines_after_imports=2
 
-# This is a first step from using the old pep8 plugin to pytest
-# We could use something else in future, like a combination linter
-# (flake8, pylint, zulint)
-[pycodestyle]
-ignore=E121,E123,E126,E226,E24,E704,W503,W504,E117,E252
+# flake8 takes a little more time than pycodestyle, but finds more things
+# (we could also look at pylint, zulint, etc.)
+[flake8]
+ignore=
+  # E121 - continuation line under-indented for hanging indent
+  E121,
+  # E123 - closing bracket does not match indentation of opening bracket's line
+  E123,
+  # E126 - continuation line over-indented for hanging indent
+  E126,
+  # E226 - Missing whitespace around arithmetic - FIXME possibly change this
```

Comment:
Is there a reason we ignore this?

Reply:
There are quite a few cases of this, and I'm hoping we can implement a general fixer tool with the possibility of pre- or post-commit hook which will fix this for us - but I wasn't sure if we actually wanted this, particularly given our line length, though we can always reformat code accordingly of course.